src/main/java/com/weather/discordweather/gateway/WeatherForecastGateway.java

```diff
@@ -86,6 +90,23 @@ public String executeWebhook(String forecast) {
         .findFirst();
   }
 
+  private String paginateForecast(String forecast) {
+    while (forecast.length() > 2000) {
+      int i = forecast.lastIndexOf('\n', 2000);
+      discordClient.executeWebhook(
+          "896866700702662697",
+          "qcvv8ihtrGTPjZswASiJaOsy-qMua58DkgAb-XA39WAG5D1FxFDz1EGJ53FavFz-GjTE",
```

Comment:
Since these show up multiple times, let's make them constants. (I realise when we get the Postgres setup working, that this won't be really used anymore, since it'll be on a per-subscription basis, but for now it'll be useful to have.)